react/features/toolbox/components/native/OverflowMenu.js

```diff
@@ -140,16 +140,15 @@ class OverflowMenu extends PureComponent<Props, State> {
                 {!toolbarButtons.has('invite') && <InviteButton { ...buttonProps } />}
                 <AudioOnlyButton { ...buttonProps } />
                 {!toolbarButtons.has('raisehand') && <RaiseHandButton { ...buttonProps } />}
-                <LobbyModeButton { ...buttonProps } />
                 <ScreenSharingButton { ...buttonProps } />
+                <SecurityDialogButton { ...buttonProps } />
```

Comment:
Let's put this where the lobby button previously was.

Reply:
Moved.